Demo/Demo/ViewModels/BaseViewModel.swift

```diff
@@ -126,6 +127,44 @@ class BaseViewModel: ObservableObject, PayPalWebCheckoutDelegate, CardDelegate {
         cardClient.approveOrder(request: cardRequest, context: context)
     }
 
+    func checkoutWithNativeClient(orderId: String?) async throws -> NativeCheckoutResult {
+        guard let orderId = orderId else {
+
+            return NativeCheckoutResult.error(CoreSDKError(code: 0, domain: "Order Id is null", errorDescription: "Order Id is null"))
+        }
+        let nativeCheckoutClient = try await getNativeCheckoutClient()
+        let paypalRequest = PayPalRequest(orderID: orderId)
+
+        class PaypalCheckoutDelegate: PayPalDelegate {
```

Comment:
Class within a class feels a little weird. I think we can just make the main class conform to PayPalDelegate or make an extension of the main class that conforms to our delegate at the end of the file and add the protocol stubs there.